PianoFromAbove/MainProcs.cpp

```diff
@@ -48,9 +48,9 @@ INT_PTR SetResolutionProc(HWND hwnd, UINT msg, WPARAM wparam, LPARAM lparam) {
         case IDOK: {
             char buf[1024] = {};
             GetWindowTextA(GetDlgItem(hwnd, IDC_WIDTH), buf, sizeof(buf));
-            int width = max(MINWIDTH, min(atoi(buf), 65535));
+            int width = max(0, min(atoi(buf), INFINITY));
```

Comment:
`INFINITY` is meant to be used as a floating point constant. I feel like what you meant was `max(0, atoi(buf));`.